apps/allocations/package-lock.json

```diff
@@ -0,0 +1,3317 @@
+{
```

Comment:
Please get rid of this file! We are not committing `package-lock` files.

Reply:
Why not just simply do:

```
echo 'package-lock=false' >> .npmrc
echo 'package-lock.json' >> .gitignore
```

I remember to have done this change before, but at some point it was reverted. I think this is a definitive solution, and even easily reversible in the remote case we want to use it again.

Reply:
Yes, that sounds like a great PR to make off of `dev`! 👍 

But for now, in this PR, let's get rid of all the `package-lock` files directly.